spec/features/edit_emails_spec.rb

```diff
@@ -0,0 +1,95 @@
+require 'rails_helper'
```

Comment:
You removed others duplicated code in `spec/features/delayed_mailer_spec.rb`, and created lots of duplicated code here.